### DISCUSSION TITLE
Archivos 404

### DISCUSSION THREAD

**Discussion by Aurorabaeza1995** [community/community#132596] [Other Feature Feedback, Questions, & Ideas]
Cómo se abren los archivos 404?`

**Comment by ImMamey** [comment]
@Aurorabaeza1995 

Los archivos con extension `.404` son una versión antigua a los archivos `.FTS`, de "FruityLoops", puedes ver mas información [aquí](https://filext.com/file-extension/404) o [aquí](https://dotwhat.net/file/extension/404/5982).

Así que supongo que puedes abrir los archivos con una versión muy muy antigua de FL studio (2003 para atrás)
